src/domain/chainContext.ts

```diff
@@ -120,7 +140,13 @@ export class ChainContext {
     );
 
     // Save the context to the static map to be used by the API
-    const context = new ChainContext(options, provider, chainId, registry);
+    const context = new ChainContext(
+      options,
+      provider,
+      chainId,
+      registry,
+      orderBookApi
+    );
```

Comment:
As `orderBookApi` is part of the `options`, would we not just use the `options` within the constructor as this is basically already passed the whole way down and we can DRY this?